hal/src/boron/network/sara_ncp_client.cpp

```diff
@@ -993,6 +1003,7 @@ int SaraNcpClient::configureApn(const CellularNetworkConfig& conf) {
 
 int SaraNcpClient::setRegistrationTimeout(unsigned timeout) {
     registrationTimeout_ = std::max(timeout, REGISTRATION_TIMEOUT);
+    LOG(INFO," Regtimeset: %u",registrationTimeout_ );
```

Comment:
Nit: formatting.

Reply:
This line should not even be a part of code review. I will try to get rid of this, but will change formatting on others.